tensorboard/plugins/graph/graphs_plugin.py

```diff
@@ -18,15 +18,21 @@
 from __future__ import division
 from __future__ import print_function
 
+import six
 from werkzeug import wrappers
 
 from tensorboard.backend import http_util
 from tensorboard.backend import process_graph
 from tensorboard.backend.event_processing import plugin_event_accumulator as event_accumulator  # pylint: disable=line-too-long
 from tensorboard.plugins import base_plugin
+from tensorboard.util import tb_logging
+
+logger = tb_logging.get_logger()
 
 _PLUGIN_PREFIX_ROUTE = 'graphs'
 
+_PLUGIN_NAME_RUN_METADATA_WITH_GRAPH = 'graph_run_metadata_graph'
```

Comment:
It took me a long time to track down that this plugin name is also
hard-coded in a core TensorFlow file that has not yet been synced out to
GitHub (Googlers, see <http://cl/234893199>). It’s confusing that we
should declare this plugin constant even though the logic that writes
this out doesn’t use the constant; I couldn’t figure out how this logic
was getting into the test cases. If this must be hardcoded, please
document exactly where else this magic string exists and how it is used.